src/components/DetailsList/DetailsHeader.tsx

```diff
@@ -31,6 +31,7 @@ export interface IDetailsHeaderProps {
   /** ariaLabel for the header checkbox that selects or deselects everything */
   ariaLabelForSelectAllCheckbox?: string;
   ref?: string;
+  hideSelectAll?: boolean;
```

Comment:
wouldn't it be more consistent to use isSelectAllVisible (like isSizing, isAllCollapsed, isAllSelected)